rules/S7511/python/metadata.json

```
@@ -0,0 +1,23 @@
+{
+  "title": "The reversing an iterable using slicing is redundant for \"set()\", \"sorted()\", or \"reversed()\" functions",
```

The title should use the "X should Y" phrasing
```suggestion
  "title": "Passing a reversed iterable to \"set()\", \"sorted()\", or \"reversed()\" should be avoided",
```